### PR TITLE
Correct Crop Rendering

### PR DESCRIPTION
**Pull request by RlonRyan**
And with that, the only thing left to get to render are the custom wood
blocks in the inventory! :yay:

Oh, and I think I found the missing ghost import...

### CODE REVIEW

File: src/main/java/com/InfinityRaider/AgriCraft/blocks/BlockBase.java [hunk 1]
```diff
@@ -113,4 +113,9 @@ public void registerIcons(IIconRegistrar iconRegistrar) {
 		this.icon = iconRegistrar.registerIcon("agricraft:blocks/" + name);
 	}
 
+	@Override
+	public EnumWorldBlockLayer getBlockLayer() {
+		return EnumWorldBlockLayer.CUTOUT;
```
Comment (InfinityRaider) [inline src/main/java/com/InfinityRaider/AgriCraft/blocks/BlockBase.java]: Maybe return the Default value you have set here instead?

**Comment by RlonRyan** [comment]
Ooops... Sorry...

File: src/main/java/com/InfinityRaider/AgriCraft/blocks/BlockBase.java (hunk 1 above)
Comment (RlonRyan) [inline src/main/java/com/InfinityRaider/AgriCraft/blocks/BlockBase.java]: What do you mean?

Comment (RlonRyan) [inline src/main/java/com/InfinityRaider/AgriCraft/blocks/BlockBase.java]: I just added back in the method I shouldn't have deleted...

Comment (InfinityRaider) [inline src/main/java/com/InfinityRaider/AgriCraft/blocks/BlockBase.java]: Its only minor, but return this default field: https://github.com/InfinityRaider/AgriCraft/blob/master/src/main/java/com/InfinityRaider/AgriCraft/blocks/BlockBase.java#L25

Comment (RlonRyan) [inline src/main/java/com/InfinityRaider/AgriCraft/blocks/BlockBase.java]: Will do!

**Pull request merged**